gdrive-userscript/cytube-google-drive.user.js

```diff
@@ -206,15 +214,15 @@ try {
         'Violentmonkey' // https://github.com/calzoneman/sync/issues/713
     ];
 
-    function isRunningTampermonkey() {
+    function isTampermonkeyCompatible() {
```

Comment:
In light of #721 I'm wondering if this shouldn't be inverted to "isn't greasemonkey" because GM is seemingly the only one so far that requires the sandboxed solution (which breaks Violentmonkey and might be responsible for that other one in #721 as well).

Not something you need to fix in this PR but getting the conversation started regardless.

Reply:
so `isntShit()`